app/services/search/base.rb

```diff
@@ -2,76 +2,43 @@ module Search
   class Base
     class << self
       def index(doc_id, serialized_data)
-        request do
-          SearchClient.index(
-            id: doc_id,
-            index: self::INDEX_ALIAS,
-            body: serialized_data,
-          )
-        end
+        Search::Client.index(
```

Comment:
basically no `request do` and `Search::Client` instead of `SearchClient`

Reply:
This reads so much better with the need to remember the `request` block 🙌